mne/io/eeglab/eeglab.py

```diff
@@ -609,9 +609,10 @@ def _read_eeglab_events(eeg, event_id=None, event_id_func='strip_to_integer'):
     if len(events) < len(types):
         warn("Some event codes could not be mapped to integers. Use the "
              "`event_id` parameter to map such events to integers manually.")
-    if len(events) < 1:
-        warn("No events found, consider adding an `event_id`. As is, the "
-             "trigger channel will consist entirely of zeros.")
+    if len(events) < 1 and len(types) > 0:
```

Comment:
right - one possibility is to move `if len(events) > 0` to a top-level if and talk about "some event codes ..." only if len(events) > 0. Another would be to remove the "no events" warnings as a special case of "some event codes could not be mapped ...".